modules/block-interactions/src/main/java/unnamed/mmo/blocks/handlers/TillHandler.java

```diff
@@ -0,0 +1,48 @@
+package unnamed.mmo.blocks.handlers;
+
+import net.kyori.adventure.sound.Sound;
+import net.minestom.server.coordinate.Point;
+import net.minestom.server.entity.Player;
+import net.minestom.server.instance.Instance;
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.instance.block.BlockHandler;
+import net.minestom.server.item.Material;
+import net.minestom.server.sound.SoundEvent;
+import net.minestom.server.utils.NamespaceID;
+import org.jetbrains.annotations.NotNull;
+import unnamed.mmo.blocks.BlockInteractionUtils;
+
+import java.util.Set;
+
+public class TillHandler implements BlockHandler {
```

Comment:
Ill have a few more comments for this when I am done with the item component stuff im working on in the item branch